src/composite-disposable.coffee

```diff
@@ -52,8 +52,9 @@ class CompositeDisposable
   #
   # * `disposable` {Disposable} instance or any object with a `.dispose()`
   #   method.
-  add: (disposable) ->
-    @disposables.add(disposable) unless @disposed
+  add: ->
```

Comment:
What about the constructor? It can be
```coffee
constructor: (disposables...) ->
  @disposables = new Set
  @add.apply disposables
```
for example